javascript/javascript2/week1/Javascript2Project/index.js

```diff
@@ -72,6 +69,17 @@ function toggleCard(event) {
 
     cardStorageArray.push(cardStorageObject);
     randomCardsArray.splice(random, 1);
+
+    let openedCards = [];
+    document.querySelectorAll(".card__info").forEach((img) => {
+      if (img.src === card.src) openedCards.push(img);
+    });
+
+    if (openedCards.length > 1) {
+      setTimeout(() => {
+        openedCards.forEach((img) => (img.style.display = "none"));
```

Comment:
This here is a good opportunity to also remove the previously added eventListener, otherwise you can cause memory leaks and performance issues - https://medium.com/@conboys111/what-happens-if-you-dont-remove-event-listeners-in-javascript-e5e2207d79ec
```suggestion
openedCards.forEach((img) => {
          img.style.display = "none";
          img.removeEventListener("click", toggleCard);
        });
```